.codespellignorewords

```diff
@@ -2,3 +2,4 @@ acn
 ACN
 inflight
 thead
+WRONLY
```

Comment:
As per the docs:
https://github.com/codespell-project/codespell/blob/master/README.rst

> The -I flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: codespell -I path/to/file.txt to execute codespell referencing said list of allowed words. Important note: The list passed to -I is case-sensitive based on how it is listed in the codespell dictionaries.

```suggestion
wronly
```

But don't feel guilty, everyone seems to do it!

Reply:
Yup, that did it!

Reply:
Phew, would have been embarrassing if it hadn't!